### PR TITLE
feature/tma 203 into develop

### PR DESCRIPTION
**Pull request by dnd-GautierBardon**
Added db_schema and data patch for product configurable attributes

### CODE REVIEW

**Comment by dnd-GautierBardon** [comment]
Added patch for api secret encryption

File: etc/db_schema.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
```
Comment (Dnd-Gimix) [inline etc/db_schema.xml:1]: Dn'D comment missing

File: Setup/Patch/Data/UpdateConfigurableAttributes.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+use Magento\Framework\Serialize\SerializerInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+/**
+ * Class UpdateConfigurableAttributes
+ *
+ * @category  Class
+ * @package   Akeneo\Connector\Setup
+ * @author    Agence Dn'D <contact@dnd.fr>
+ * @copyright 2010-present Agence Dn'D
```
Comment (Dnd-Gimix) [inline Setup/Patch/Data/UpdateConfigurableAttributes.php:20]: 2004

Comment (Dnd-Gimix) [reply Setup/Patch/Data/UpdateConfigurableAttributes.php]: ok

File: etc/db_schema.xml (hunk 1 above)
Comment (Dnd-Gimix) [reply etc/db_schema.xml:1]: ok

**Pull request merged**